detect_secrets/plugins/jwt.py

```diff
@@ -0,0 +1,53 @@
+"""
+This plugin finds JWT tokens
+"""
+from __future__ import absolute_import
+
+import base64
+import json
+import re
+
+from .base import RegexBasedDetector
+
+try:
+    # Python 2
+    from future_builtins import filter
+except ImportError:
+    # Python 3
+    pass
+
+
+class JwtTokenDetector(RegexBasedDetector):
+    secret_type = 'JSON Web Token'
+    denylist = [
+        re.compile(r'eyJ[A-Za-z0-9-_=]+\.[A-Za-z0-9-_=]+\.?[A-Za-z0-9-_.+/=]*'),
```

Comment:
Can you add a `?` after the `*`? I believe this will make it non-greedy i.e. more efficient.

Reply:
@KevinHock Absolutely, and thanks for pointing it out.